packages/blade/src/components/Button/BaseButton/BaseButton.tsx

```diff
@@ -73,17 +187,23 @@ const BaseButton = ({
       }}
       disabled={isDisabled}
       activeColor={activeColor}
+      buttonHeight={buttonHeight}
+      buttonSpacing={buttonSpacing}
     >
-      {Icon ? <Icon size="small" color="action.icon.primary.default" /> : null}
-      <BaseText
-        lineHeight="s"
-        fontSize={100}
+      {Icon && iconPosition == 'left' ? <Icon size={iconSize} color={iconColor} /> : null}
+      <StyledBaseText
+        lineHeight={lineHeight}
+        fontSize={fontSize}
```

Comment:
TS doesn't give any hints here as to what scale represent what size in `px/rem`. This makes the dev experience difficult because now we need to open up the `typography.ts` file and check what scale is what value. Figma also doesn't help here since it shows the font size directly in pixel and not the scale we made. This issue stands true on `BaseText.stories.tsx` as well

<img width="665" alt="Screenshot 2022-05-25 at 4 46 15 PM" src="https://user-images.githubusercontent.com/24487274/170250043-bb045efc-c3fe-4b38-b53f-3ac3f6f9b1a1.png">

Reply:
`StyledBaseText` is `BaseText` wrapped with styled. But even if its not wrapped with style, the same issue persists

Reply:
Damn. Huge bummer. This would've been very helpful

Reply:
> Unions can't have jsdoc descriptions. it's keyof nah..

@anuraghazra didn't get this? 🤔